docs/api-tokens.md

```diff
@@ -0,0 +1,111 @@
+# API Tokens
+
+Terraform maakt gebruik van verschillende API tokens.
+
+## GitHub Fine-grained Personal Access Token (PAT) genereren
+
+1. Ga bij GitHub naar [Developer Settings/Personal access tokens][github-developer-settings] en log in als dat nodig is.
+1. Maak een nieuw token aan (het kan zijn dat je (nogmaals) met 2-factor authenticatie moet bevestigen):
+   - Geef het token een zinnige naam, bijvoorbeeld `NLDS Terraform` voor cloud-gebruik of `NLDS Terraform local` voor
+     lokaal gebruik op de command line.
+   - Kies `nl-design-system` onder ‘Resource owner’.
+   - Geef het token toegang tot `All repositories`.
+   - Geef het token de volgende ‘Repository permissions’:
+     - **Read** access tot 'Metadata' (deze permissie is default en niet aan te passen).
+     - **Read** and **Write** access tot ‘Administration’, ‘Code’, en ‘Pages’.
+   - Geef het token de volgende ‘Organization permissions’:
+     - **Read** access tot ‘Administration’.
+     - **Read** and **Write** access tot ‘Members’.
+1. Een custom expiration van 1 dag is voor lokaal gebruik waarschijnlijk voldoende omdat hetzelfde token eenvoudig
+   opnieuw te genereren is met een nieuwe expiration date. Voor het token dat door HCP Terraform (de cloud omgeving)
+   gebruikt wordt, is een token met een geldigheid van een jaar een goede keuze.
+1. Kopieer het token.
+
+## Vercel API token genereren
+
+1. Ga bij Vercel naar [Account Settings/Tokens][vercel-settings-tokens] en log in als dat nodig is.
+1. Maak een nieuw token aan:
+   - Geef het een zinnige naam, bijvoorbeeld `NLDS Terraform` voor cloud-gebruik of `NLDS Terraform local` voor lokcaal
+     gebruik op de command line.
+   - Selecteer `nl-design-system` als scope.
+   - Gebruik een zinvolle expiration date. Voor lokaal gebruik is 1 dag vaak voldoende. Voor HCP Terraform (de cloud
+     omgeving), is een token met een geldigheid van een jaar een goede keuze.
+1. Kopieer het token.
+
+## API tokens gebruiken bij HCP Terracloud
+
+1. Ga bij HCP Terraform naar [Workspaces/Variables][terraform-workspaces-variables]
+1. Werk het betreffende token `VERCEL_API_TOKEN`, `GITHIB_TOKEN`, of `PLESK_UUID_TOKEN` bij.
+   - Plak onder `value` het token dat je gekopieerd hebt.
+   - Werk de description bij zodat duidelijk is waar en door wie het token gegenereerd is en hoe lang het gebruikt kan
+     worden.
+
+## API tokens gebruiken op de command line
+
+Zorg dat je in je browser bent ingelogd bij [terraform](https://app.terraform.io).
+
+In je terminal moet je ook inloggen, dat doe je zo.
+
+```shell
+$ terraform login
```

Comment:
Zou het handig zijn om de input en de output op te splitsen in losse blocks, en de `$` te verwijderen uit de input, zodat het makkelijker te kopieren is? Kunnen kopieren lijkt me handig, zeker als die spatie aan het begin belangrijk is.

Reply:
Ik ga het proberen. Ik had er hier vanaf geweken omdat precies hier in/output wel een beetje door elkaar staat.